54 model program

Castronova opened the pull request:
Implemented the new workflow.

mjstealey commented:
@Castronova - This pull request still retains items that appear to be personal environment variables that should not be pulled into the main develop repository. Also some doc files that seem no relevant to committing to the primary develop branch. Are the 146 files listed in this pull request actually all required? or can some be eliminated due to either personal or demo info nature.

Castronova commented:
@mjstealey I didn't realize how many extra files I was committing. I cleaned up my branch and also reverted local changes to fig.yml, settings.py, and urls.py.  Please let me know if there is anything that I missed.
